pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pelican-seo"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
Your previous commit already changed this in current master: https://github.com/pelican-plugins/seo/blob/master/pyproject.toml#L3

In the system we are using, version numbers are not meant to be manually modified this way. AutoPub will automatically increment the version number as instructed by the `[major|minor|patch]` designation included in the `RELEASE.md` file. Ergo, this line should be changed to set the version back to `1.0.0` — otherwise version 1.0.2 will be published, and version 1.0.1 will end up being skipped, which isn't what we want to happen.

Reply:
Ok, thanks for the explanation. In this case I'll merge this PR as is, and push a new PR that revert the version and incluse the `RELEASE.md`.